doc/source/io.rst

```diff
@@ -4548,16 +4548,15 @@ Read from a parquet file.
 
 .. ipython:: python
 
-   result = pd.read_parquet('example_pa.parquet', engine='pyarrow')
    result = pd.read_parquet('example_fp.parquet', engine='fastparquet')
+   result = pd.read_parquet('example_pa.parquet', engine='pyarrow')
```

Comment:
I also switched the order here, as it is the last one for which the dtypes are shown, and `pyarrow` preserves timezones while `fastparquet` does not, and it is nice to show this capability of `pyarrow`)

Reply:
Removed this change here, left this for https://github.com/pandas-dev/pandas/pull/18662, so this PR can more easily be reverted.